packages/project-tee-starter/e2e/starter-plugin.test.ts

```diff
@@ -199,7 +209,7 @@ export class StarterTestSuite implements TestSuite {
 
           if (
             service.capabilityDescription !==
-            'This is a starter service which is attached to the agent through the starter plugin.'
+            'This is a starter service, can be customized for Mr. TEE.'
```

Comment:
We can also remove the Service test since there isn't a TEE Simulator running to call the TEE functions